backend/drtrottoir/management/commands/mockdata.py

```diff
@@ -184,7 +184,23 @@ def handle(self, *args, **options):
         for s in schedule_assignments:
             s.save()
 
-        self.stdout.write("Adding schedule work entries...")
+        self.stdout.write("Adding garbage types...")
+        types = [
+            "Rest",
+            "GFT",
+            "Glas",
+            "PMD",
+            "Klein Gevaarlijk Afval",
+            "Nucleair Afval",
+        ]
+        garbage_types = [GarbageType(name=t) for t in types]
+
+        for x in garbage_types:
+            x.save()
+
+        self.stdout.write(
```

Comment:
Is also just a print statement right? Why is this happening in such a non pythonic way?